dts/bindings/spi/andestech.atcspi200.yaml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2019, Andes Technology Corporation.
```

Comment:
s/2019/2022

Reply:
Thanks for reviewing.

I have fixed this incorrect date.